src/openforms/js/components/admin/forms/Inputs.js

```diff
@@ -66,7 +68,22 @@ const DateTimeInput = ({name, value, formatDatetime, onChange, ...extraProps}) =
 
   return (
     <div className="datetime-input">
-      <input ref={datetimePickerRef} placeholder={placeHolder} />
+      <input
+        ref={datetimePickerRef}
+        placeholder={placeHolder}
+        value={value ? value : ''}
```

Comment:
```suggestion
        value={value ?? ''}
```
this will use the RHS if `value` is `undefined` or `null`, and otherwise use `value`. Small stylistic thing, not an essential change!